tests/unit/backend/test_common.py

```diff
@@ -45,3 +45,97 @@ class TestBackend(PyRDF.backend.Backend.Backend):
 
         with self.assertRaises(TypeError):
             TestBackend()
+
+class IncludeHeadersTest(unittest.TestCase):
+    """
+    Tests to check the working of 'PyRDF.Include' function.
+
+    """
+    class TestBackend(PyRDF.backend.Backend.Backend):
+        """
+        A Test Backend class to check the working of
+        'PyRDF.Include' function.
+
+        """
+        def execute(self, generator):
+            pass
+
+    def test_default_empty_list_include(self):
+        """
+        Test case to ensure that 'PyRDF.Include' function
+        defaults to an empty list if no parameter is given.
+
+        """
+        PyRDF.current_backend = IncludeHeadersTest.TestBackend()
+
+        PyRDF.include()
```

Comment:
I would not allow this case, no default value for the list

Reply:
I'll remove it @etejedor 👍